es/pck.tex

```diff
@@ -4,76 +4,75 @@ \chapter{Conocimiento de la pedagogía del contenido}\label{s:pck}
+  el conocimiento de la pedagogía del contenido incluye qué ejemplos usar cuando se enseña, cómo se incluyen parámetros en una función o qué conceptos erróneos sobre etiquetas HTML anidadas son los más comunes.
 \end{description}
 
 Podemos agregar conocimiento técnico a este conjunto~\cite{Koeh2013},
-pero eso no cambia el punto clave: no es suficiente saber sobre el tema y como enseñar---tienes que saber cómo enseñar ese tema en particular~\cite{Maye2004}.
-Este capítulo resume algunos resultados de investigaciones sobre enseñanza de informática para añadir a tu colección de PCK.
+pero eso no cambia el punto clave: no es suficiente saber sobre el tema y cómo enseñar---tienes que saber cómo enseñar ese tema en particular~\cite{Maye2004}.
+Este capítulo resume algunos resultados de investigaciones sobre enseñanza de informática para añadir a tu colección sobre el conocimiento de la pedagogía del contenido.
 
 Como con toda investigación,
 se requiere cierta precaución al interpretar los resultados:
 
 \begin{description}
 
 \item[Las teorías cambian a medida que se obtienen más datos.]
-  La investigación en educación en informática (CER, por Computing education research en inglés) es una disciplina nueva:\index{computing education research}
-  la Sociedad Americana de Educación en Ingeniería fue fundada en 1893 y el Consejo Nacional de Profesores de Matemática en 1920, pero la Asociación de Profesores de Informática no se creó hasta 2005.
+  La investigación sobre educación en informática es una disciplina nueva:\index{computing education research}
+  la Sociedad Americana de Educación en Ingeniería fue fundada en 1893 y el Consejo Nacional de Docentes de Matemática en 1920, pero la Asociación de Docentes de Informática no se creó hasta 2005.
   Mientras que existe un flujo constante de nuevo conocimiento en conferencias como \hreffoot{http://sigcse.org/}{SIGCSE},
-  \hreffoot{http://iticse.acm.org/}{ITiCSE},
-  y \hreffoot{https://icer.hosting.acm.org}{ICER},
-  simplemente no sabemos tanto sobre como aprender a programar como sí sabemos sobre aprender a leer, jugar un deporte o resolver cálculos simples.
+  \hreffoot{http://iticse.acm.org/}{ITiCSE}
+  e \hreffoot{https://icer.hosting.acm.org}{ICER},
+  simplemente no sabemos tanto sobre cómo aprender a programar como sí sabemos sobre aprender a leer, jugar un deporte o resolver cálculos simples.
  
-\item[La mayoría de las personas en estos estudios son WEIRD:]\index{WEIRD}
-  viven en sociedades occidentales, democráticas, industrializadas y con alto nivel de riqueza y educación (WEIRD por Western, Education, Industrialized, Rich, and Democratic en inglés)~\cite{Henr2010}.
+\item[La mayoría de las personas en estos estudios
+  viven en sociedades occidentales, democráticas, industrializadas y con alto nivel de riqueza y educación], tal como señala ~\cite{Henr2010}.
```

Comment:
Hay una referencia la index que se pierde si sacamos WEIRD, que también es un juego de palabras porque significa raro, extraño.  De nuevo, no me gusta remover todas las siglas si son conceptuales y es la forma en la que uno lo va a encontrar en otra bibliografía (pero puedo estar muy sesgada por los textos de informática/ciencias de la computación)

Creo que agregaría el siguiente texto luego de la palabra educación y antes del corchete (por qué usa corchetes?....voy a ver como queda en el PDF): Se los denomina WEIRD\index{WEIRD} por Western, Education, Industrialized, Rich, and Democratic en inglés, tal como señala.